src/sync.cpp

```diff
@@ -38,7 +38,7 @@ namespace avt_vimba_camera {
 Sync::Sync(ros::NodeHandle nh, ros::NodeHandle nhp): nh_(nh), nhp_(nhp), init_(false), it_(nh)
 {
   // Read params
-  nhp_.param("camera", camera_, string("/stereo_down"));
+  nhp_.param("camera", camera_, string("/stereo"));
```

Comment:
Should it be defaulted to something else instead of stereo since the PR is to remove the stereo camera code?